examples/sri-1/components/workgroup/workstep/workstep.ts

```diff
@@ -0,0 +1,71 @@
+import { IWorkstep } from "./workstep.interface";
+import { Agreement } from '../storage/agreement';
+import { Security } from "../policy/security";
+import { Privacy } from "../policy/privacy";
+import { WorkgroupComponent } from "../workgroup/workgroupComponent.service";
+
+export class Workstep implements IWorkstep {
+    name: string;
+    id: string; //[R217] A workstep MUST have a unique identifier within a BPI.
+    workgroupId: string; //[R215] A workstep instance MUST be associated with only one workgroup.
+    version: string; //[R220] A workstep MUST be versioned within a BPI.
+    status: string; //[R219] A workstep instance MUST NOT be updated while the workstep is being executed by the BPI.
+    businessLogicToExecute: any; // [R211] A workstep MUST have an input, one or more process steps, and an output. This is just a well-known convention from business process management frameworks.
```

Comment:
This was just a hacky way to implement processing in the monolith test suite. Workstep execution will be a completely different mechanism in the SRI, so i would keep it out for now.

Reply:
removed this mock.